app/Party.php

```diff
@@ -62,6 +63,7 @@ public function findAll()
                     `e`.`free_text`,
                     `e`.`hours`,
                     `e`.`wordpress_post_id`,
+                    `e`.`discourse_thread`,
```

Comment:
Out of interest, I wonder if some/all of these old DB::raw queries are dead code now.

Reply:
I wondered that too, but I was feart.

Reply:
Actually quick scan says I think they might be being used.  Ho hum.  Might be a nice one for a little coverage-increasing and refactoring if I'm bored of an eve.